package.json

```diff
@@ -82,7 +81,7 @@
     "vscode-proxy-agent": "^0.8.2",
     "vscode-regexpp": "^3.1.0",
     "vscode-ripgrep": "^1.11.1",
-    "vscode-sqlite3": "4.0.10",
+    "vscode-sqlite3": "microsoft/vscode-node-sqlite3#cfa67632187259672624267ded08a2e1f7a81955",
```

Comment:
Should I publish a new version?

Reply:
Sure, changes are in this branch https://github.com/microsoft/vscode-node-sqlite3/tree/vscode-electron-12

Reply:
This was a bit more work because the current `vscode` branch is already way ahead, so I decided to create a new `vscode-no-napi` branch to ensure we are not using NAPI by accident. I also cherry-picked all the commits I did to enable GitHub CI in that branch and increased the version to `4.0.11`.

But now it looks like the CI is failing, and I do not understand that: https://github.com/microsoft/vscode-node-sqlite3/runs/2303833558

```
Error: The module '/Users/runner/work/vscode-node-sqlite3/vscode-node-sqlite3/build/Release/sqlite.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 85. This version of Node.js requires
NODE_MODULE_VERSION 83. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1122:18)
```

Reply:
Yup thats because you had built the module for electron runtime but the tests are run on node runtime, `yarn test` uses node binary.

Reply:
Done.